src/webapp/forms.py

```diff
@@ -334,6 +346,7 @@ def __init__(self, *args, **kwargs):
         self.pi_department_or_organization.data = kwargs.get("pi_department_or_organization", self.pi_department_or_organization.data)
         self.pi_institution.data = kwargs.get("pi_institution", self.pi_institution.data)
         self.field_of_science.data = kwargs.get("field_of_science", self.field_of_science.data)
+        self.field_of_science_id.data = kwargs.get("field_of_science_new", self.field_of_science_id.data)
```

Comment:
```suggestion
        self.field_of_science_id.data = kwargs.get("field_of_science_id", self.field_of_science_id.data)
```
? I don't see field_of_science_new elsewhere in the new code.